### PR TITLE
Assignment2 @Disciple4GIT

### PR DESCRIPTION
**Pull request by A-Tanner**
[Link to Page](https://gray-wave-023e0701e.1.azurestaticapps.net)

### CODE REVIEW

**Comment by jphillips16** [comment]
Great work! Seems you hit all of the requirements. One thing of note, when you change the color of the letters, its hard to know what the specific colors mean. Like it was difficult for me to know when I put the letter in the right spot or if it was the right letter in the wrong spot. Other than that, I loved the way you displayed the guesses and the fact you used an actual hamburger.

File: wordle-web/components/settings-dialog.vue [hunk 1]
```diff
@@ -1,74 +1,119 @@
...
-            </v-menu>
-          </v-card-text>
-        </v-container>
-      </v-card>
-    </v-dialog>
+    <div>      
+        <v-btn
+          icon @click ="toggleMenu">
+          <v-icon> mdi-hamburger</v-icon>
+        </v-btn>
```
Comment (mmwoodfo) [inline wordle-web/components/settings-dialog.vue:6]: I would suggest putting this in it's own component, outside of the settings dialog.

File: wordle-web/components/settings-dialog.vue [hunk 2]
```diff
@@ -1,74 +1,119 @@
...
+              <v-card-text>
+                <v-menu offset-y>
+                  <template #activator="{ on, attrs }">
+                      <v-btn color="primary" v-bind="attrs" v-on="on" nuxt to="/about" @click="menu=!menu"> About Page </v-btn>
+                  </template>
+                </v-menu>
+              </v-card-text>
+            </v-container>
+          </v-card>
+        </v-dialog>
```
Comment (mmwoodfo) [inline wordle-web/components/settings-dialog.vue:26]: This too, should go in it's own component. I would also have suggested making this either a navigation drawer or a v-menu, not a dialog, as that's not a typical form of navigation.

File: wordle-web/components/keyboard.vue [hunk 3]
```diff
@@ -77,5 +90,23 @@ export default class KeyBoard extends Vue {
 
     return Letter.getColorCode(LetterStatus.Unknown)
   }
+
+  clickNoise(){
```
Comment (mmwoodfo) [inline wordle-web/components/keyboard.vue:94]: Very cool with the audio! Good job!

**Pull request closed without merging**